include/CL/cl.h

```diff
@@ -1375,6 +1375,18 @@ clEnqueueTask(cl_command_queue  /* command_queue */,
               const cl_event *  /* event_wait_list */,
               cl_event *        /* event */) CL_EXT_SUFFIX__VERSION_1_2_DEPRECATED;
 
+/* Poclcc OpenCL APIs */
+extern CL_API_ENTRY cl_int CL_API_CALL
+clExportPoclccBinary(cl_program /* program */,
```

Comment:
What is the purpose of these new API functions? First, we cannot add them to cl.h nor with direct cl-prefix (those are preserved for official OpenCL APIs), second, why don't the clGetProgramInfo() suffice?